docs/guides/websites/hosting/secure-website-lets-encrypt-acme-sh/index.md

```diff
@@ -187,6 +213,10 @@ The target directory must exist first. To create it, run:
     mkdir -p /etc/ssl/example.com
 {{< /note >}}
 
+{{< note >}}
```

Comment:
Added a caveat here because the file system locations changed from earlier in the guide, and some users might copy/paste this without checking for that difference